R/snapshot.R

```diff
@@ -134,7 +134,8 @@ snapshot_lines <- function(x, transform = NULL) {
   if (!is.null(transform)) {
     x <- transform(x)
   }
-  x <- indent(x)
+  # if transform() wiped out the full message, don't indent, #1487
+  if (length(x)) x <- indent(x)
```

Comment:
I think it would be better to fix this in indent, so that `indent(character())` returns `character()`. That preserves the invariant that the `indent()` returns a character vector the same length as the input.

(The root cause is that `paste()`'s recycling rules aren't quite what we want here)